ledger/test-common/src/main/daml/performance/PingPongExplode.daml

```diff
@@ -0,0 +1,146 @@
+-- Copyright (c) 2020 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+-- SPDX-License-Identifier: Apache-2.0
+
+daml 1.2
+module PingPongExplode where
+
+import DA.Text
+import PingPong
+
+template PingExplode
+  with
+    ping : Ping
+    maxLevel : Int
+  where
+
+    signatory ping.initiator
+    observer ping.responders
+
```

Comment:
you don't need that for this performance test. you could just remove the explode, merge and collapse contracts.

Reply:
Yes, @SamirTalwar-DA and I discussed removing the extra DAML but there's a chance we'll soon add more complex benchmarks based on Explode/Cycle so we settled on splitting the code and leaving it there for the moment.

Reply:
If not happening in a couple of weeks max, I'll remove it.

Reply:
#5652